api4/config.go

```diff
@@ -107,6 +107,18 @@ func updateConfig(c *Context, w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	LockTeammateValChanged := !*appCfg.TeamSettings.LockTeammateNameDisplay && *cfg.TeamSettings.LockTeammateNameDisplay
```

Comment:
this line is confusing me, why are we requiring one side to be negated while the other doesn't have to be negated? I don't get the info either from the name of the variable storing the result nor do I get it from the same name of the setting.

Reply:
+1 to the above, also style nit: we use camel case for local variables so `LockTeammateValChanged` -> `lockTeammateValChanged`

Reply:
Fixed. Thanks!